### ISSUE TITLE
[Bug] Dependency issue when installing @temporalio/interceptors-opentelemetry v1.9.0

### ISSUE THREAD

**Issue by domarp-j** [temporalio/sdk-typescript#1350]
### What are you really trying to do?

I am attempting to update the following packages from 1.8.6 to 1.9.0:

```
"@temporalio/activity": "1.8.6",
"@temporalio/client": "1.8.6",
"@temporalio/interceptors-opentelemetry": "1.8.6",
"@temporalio/testing": "1.8.6",
"@temporalio/worker": "1.8.6",
"@temporalio/workflow": "1.8.6",
```

### Describe the bug

I can update every package except interceptors-opentelemetry without issue. When I try to bump I-OT, I see:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: temporal-hello-world@0.1.0
npm ERR! Found: @temporalio/activity@1.9.0
npm ERR! node_modules/@temporalio/activity
npm ERR!   @temporalio/activity@"1.9.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @temporalio/activity@"file:../activity" from @temporalio/interceptors-opentelemetry@1.9.0
npm ERR! node_modules/@temporalio/interceptors-opentelemetry
npm ERR!   @temporalio/interceptors-opentelemetry@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /home/domarp/.npm/_logs/2024-02-05T19_44_29_102Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /home/domarp/.npm/_logs/2024-02-05T19_44_29_102Z-debug-0.log
```

### Minimal Reproduction

1. Pull down or fork https://github.com/domarp-j/temporaltest
2. `npm i`
3. `npm i @temporalio/interceptors-opentelemetry`

### Environment/Versions

- OS and processor: x86 Windows, WLS2
- Temporal Version: Temporal 1.9.0
- Node v20.10.0
- npm v10.2.3
- Are you using Docker or Kubernetes or building Temporal from source?: No

### Additional context

Related to https://github.com/temporalio/sdk-typescript/pull/1330.

**Comment by bergundy** [comment]
I published 1.9.1 a couple of hours ago, it should be fixed now.
Please confirm that it's fixed and close this issue.

**Comment by domarp-j** [comment]
@bergundy Confirmed that installing 1.9.1 of the following packages does indeed work:

```
"@temporalio/activity": "1.9.1",
"@temporalio/client": "1.9.1",
"@temporalio/interceptors-opentelemetry": "1.9.1",
"@temporalio/testing": "1.9.1",
"@temporalio/worker": "1.9.1",
"@temporalio/workflow": "1.9.1",
```

Specifically, I was able to install `interceptors-opentelemetry` without running into those dependency issues.

Thank you for resolving this quickly!
